src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,48 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.5.0
+14 February, 2023
+
+### Breaking Change
+
+- Create dedicated registry serviceAccounts by default
+
+### Features
```

Comment:
The is also support for network policies templates

Reply:
I dont understand this, can you elaborate more? You mean a new feature that is missing or is it related with the dedicated registry sa?